eigen/examples/svd/main/main.cpp

```diff
@@ -0,0 +1,57 @@
+/* Import Prebuilt Library Example
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <esp_log.h>
+
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include <iostream>
```

Comment:
Since you also `printf` below, do you need iostream? Note that usage of iostream increases the binary size quite significantly, not sure if we should use that in the examples.

https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/cplusplus.html#iostream

Reply:
iostream used for print formated matrix values. I can't remove it.

Reply:
Alright, that makes sense. In that case, could you modify `printf("Eigen example.\n");` to also use std::cout?